geant4.sh

```diff
@@ -75,21 +86,30 @@ proc ModulesHelp { } {
 set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
 module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
 # Dependencies
-module load BASE/1.0
+module load BASE/1.0 ${XERCESC_VERSION:+xercesc/$XERCESC_VERSION-$XERCESC_REVISION}
```

Comment:
Please use  `${XERCESC_REVISION+` rather than `${XERCESC_VERSION` to be consistent with the rest of alidist.

Reply:
Done